groups/bsl/bslstl/bslstl_optional.h

```diff
@@ -1769,73 +1758,55 @@ class optional {
 
     optional(bsl::allocator_arg_t, allocator_type allocator);
         // Create a disengaged 'optional' object.  Use the specified
-        // 'allocator' to supply memory for future objects.
+        // 'allocator' to supply memory.
 
     optional(bsl::allocator_arg_t, allocator_type allocator, bsl::nullopt_t);
         // Create a disengaged 'optional' object.  Use the specified
-        // 'allocator' to supply memory for future objects.
+        // 'allocator' to supply memory.
 
-    optional(bsl::allocator_arg_t, allocator_type allocator, const optional&);
-        // If specified 'rhs' contains a value, initialize the contained
-        //  'value_type' object with '*rhs'.  Otherwise, create a disengaged
-        // 'optional'.  Use the specified 'allocator' to supply memory.
+    optional(bsl::allocator_arg_t,
+             allocator_type       allocator,
+             const optional&      original);
+        // If specified 'original' contains a value, initialize the contained
+        // 'value_type' object with '*original'.  Otherwise, create a
+        // disengaged 'optional'.  Use the specified 'allocator' to supply
+        // memory.
```

Comment:
`Create an 'optional' object ...`

Reply:
Done. Modified all relevant instances of "'optional'" to "'optional' object" too.